docs/developer-docs/use-cases/considerations-for-nft-devs.md

```diff
@@ -1,104 +1,44 @@
+The Internet Computer Protocol brings a lot of potential for NFTs. 
 
-There are dedicated services on ICP to keep an audit log of transactions such as [CAP](https://cap.ooo/), which can be used by an NFT collection as a service. This allows simple integration of the provenance history in explorers and wallets. 
-Furthermore, the state of ownership could be reconstructed in case the main NFT canister gets lost. However, some drawbacks have to be considered, e.g. NFT transfers incur additional costs due to the necessary inter-canister calls. 
+### NFT asset storage for cross-chain NFTs
 
-## Advanced topics
+NFT assets, including images, sound clips, or videos, can be securely stored on-chain entirely on ICP, even while the NFT itself is minted on various chains, such as on any EVM chain.
 
-### Think about governance
+Refer to [this example](https://github.com/domwoe/erc-721-ic-assets) on how to host metadata on the ICP of an ERC-721 NFT minted on an EVM chain.
 
-The value proposition of most NFTs is their permanence and immutability, e.g. by setting the [black hole canister](https://github.com/ninegua/ic-blackhole) as the only controller. As long as NFT canisters have their developers as controllers, users depend on the trustworthiness (and operational security) of the developers. Developers should therefore make the canisters immutable or manage the canisters with a DAO. A middle ground are mechanisms like [Launchtrail](https://devpost.com/software/launch-trail) that make changes to a canister auditable.
+### Generative NFTs 
 
-Blackholing a canister has its issues as well. If there are bugs in the canister code or you’re using experimental system APIs that might get deprecated, later on, the canister might stop functioning. 
+Dynamic NFTs can be curated to evolve based on both data stored on ICP or external APIs. NFT assets can be modified in real-time by calling API requests on-chain using [HTTPS outcalls](/https-outcalls).
 
-More information on this topic can be found in the [trust in canisters](/concepts/trust-in-canisters.md) article.
+### Minting NFTs on ICP
 
+For developers seeking to store assets and mint NFTs directly on the Internet Computer Protocol (ICP), an NFT implementation typically encompasses three core functionalities:
 
-### Think about economic sustainability
+- A registry to monitor ownership and facilitate transfers.
+- A ledger or transaction history.
+- The actual asset itself (in the case of digital assets).
 
-Ideally, your canisters implement mechanisms to generate fees that the canisters can use to pay for their existence indefinitely. A simple approach is to utilize (parts of) the transfer fee to fuel the canisters, but more elaborate schemes could involve staking or other advanced mechanisms. We’re unaware of any good best practices, but please share if you know of projects implementing clever mechanisms.
+#### NFT interface specifications and implementations
```

Comment:
We list DIP-721 and ICRC-7 here. Let's try to explain the difference between then and what we recommend to use?

Reply:
I have to make myself more well-versed on the two subject matters. I will have the capacity to add it by Friday. If it is okay with you, I will open a new PR related to the differences by Friday!